sql_scripts/11_beers_q_filter.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO beer.beers_quarantine (id)
```

Comment:
You would need to improve this query a bit, because once you run this script second time it will grab the same ID's that were inserted during first run, and you will receive constraint violation error cause id field in beer.beers_quarantine is PRIMARY KEY. To fix this you would need to upsert values in this table not just insert, add a ON CONFLICT behaviour during INSERT